lib/web_ui/lib/src/engine/canvaskit/surface.dart

```diff
@@ -17,6 +17,11 @@ import 'initialization.dart';
 import 'surface_factory.dart';
 import 'util.dart';
 
+// Only supported in profile/release mode. Allows Flutter to use MSAA but
+// removes the ability for disabling AA on Paint objects.
+const bool _kUsingMSAA =
+    bool.fromEnvironment('flutter.web.msaa', defaultValue: false);
```

Comment:
nit: should we have canvaskit here since this won't apply to HTML backend for now?

Reply:
Done